pandas/tests/indexing/test_partial.py

```diff
@@ -637,3 +637,46 @@ def test_partial_set_empty_frame_empty_consistencies(self):
         df.loc[0, 'x'] = 1
         expected = DataFrame(dict(x=[1], y=[np.nan]))
         tm.assert_frame_equal(df, expected, check_dtype=False)
+
+    def test_access_timezoned_datetimeindex_with_timezoned_label(self):
+
+        # GH 6785
+        # timezone was ignored when string was provided as a label
+
+        first_january = pd.date_range('2016-01-01T00:00', '2016-01-01T23:59',
+                                      freq='T', tz="UTC")
+        df = pd.DataFrame(index=first_january, data=np.arange(len(
+                          first_january)))
+
+        former_naive_endpoint_df = df[
```

Comment:
can you use result= and expected= here (and then a comment if needed)

Reply:
that's simple and done